new/README.md

```diff
@@ -0,0 +1,34 @@
+# yet Another Gamma Index Tool
+
+Library written in C++ for performing efficient comparisons of DICOM images using 2D, 2.5D and 3D gamma index concept.
+
+## Installation
+
+Build steps:
+1. Creating directory with built library
+```
+mkdir build && cd build
```

Comment:
See https://github.com/DataMedSci/gi-yagit/pull/52/files#diff-faff1af3d8ff408964a57b2e475f69a6b7c7b71c9978cccc8f471798caac2c88

Reply:
addressed in my other comment :)